susemanager/src/mgr_bootstrap_data.py

```diff
@@ -682,4 +710,14 @@
         'BASECHANNEL' : 'centos7-x86_64', 'PKGLIST' : RES7,
         'DEST' : '/srv/www/htdocs/pub/repositories/centos/7/bootstrap/'
     },
+    'ubuntu-16.04-amd64' : {
+        'BASECHANNEL' : 'ubuntu-16.04-amd64-main', 'PKGLIST' : PKGLISTUBUNTU,
```

Comment:
We have product ids for Ubuntu. We could use the standard configuration similar to SLE15 .
But we need to call it allways with `--with-custom-channels`. So I am a bit unsure what type is better.